admin/static/js/metrics/metrics.js

```diff
@@ -236,7 +237,11 @@ var getWeeklyUserGain = function() {
 
 };
 
-var renderKeenMetric = function(element, type, query, height, colors=null, keenClient=client) {
+var renderKeenMetric = function(element, type, query, height, colors, keenClient) {
+
+    if (!keenClient) {
```

Comment:
It does indeed work! There are lots of places where this is called without specifying a client. Maybe didn't even have to change it as I changed it to start to accommodate older safari, but there ya go....